docs/skopeo-sync.1.md

```diff
@@ -68,6 +68,10 @@ Print usage statement.
 
 **--append-suffix** _tag-suffix_ String to append to destination tags.
 
+**--digestfile** _path_
+
+After copying the image, write the digest of the resulting image to the file.
```

Comment:
The description needs updating a bit, there is no single “the resulting image”.

---

This should at least briefly document the format of the output, whatever the format ends up being.